calicoctl/calico_ctl/node.py

```diff
@@ -348,25 +349,24 @@ def node_start(node_image, runtime, log_dir, ip, ip6, as_num, detach,
 
     if runtime == 'docker':
         _start_node_container_docker(ip, ip6, as_num, log_dir, node_image, detach,
-                                     etcd_envs, etcd_volumes, etcd_binds, no_pull, backend)
-        if libnetwork_image:
-            _start_libnetwork_container(libnetwork_image, etcd_envs,
-                                        etcd_volumes, etcd_binds, no_pull)
+                                     etcd_envs, etcd_volumes, etcd_binds, libnetwork_enabled, no_pull, backend)
+
     if runtime == 'rkt':
         _start_node_container_rkt(ip, ip6, as_num, node_image, etcd_envs,
                                   etcd_volumes, etcd_binds, backend)
 
 
 def _start_node_container_docker(ip, ip6, as_num, log_dir, node_image, detach, etcd_envs,
-                                 etcd_volumes, etcd_binds, no_pull, backend):
+                                 etcd_volumes, etcd_binds, libnetwork_enabled, no_pull, backend):
     """
     Start the main Calico node container.
 
     :param ip:  The IPv4 address of the host.
     :param ip6:  The IPv6 address of the host (or None if not configured)
     :param as_num: The AS number for the host
     :param log_dir:  The log directory to use.
-    :param node_image:  The calico-node image to use.
+    :param libnetwork_enabled: True to run libnetwork plugin inside calico-node.
+    False by default. None, if not using libnetwork.
```

Comment:
This line is bad. You've already said it's defaulted to false at the point where it's defaulted. It's wrong to repeat that here.